tasks/configuration.yml

```diff
@@ -3,16 +3,16 @@
 
 - name: list of required SAP Notes
   debug:
-    var: sap_preconfigure_sapnotes | difference([''])
+    var: __sap_preconfigure_sapnotes | difference([''])
```

Comment:
I notice that you often use this idiom `listvar | difference([''])` - what is the intention here?

Reply:
I will ask the author who "invented" this in the current and previous versions of the RHEL System Roles for SAP.

Reply:
I think I found it. We want to avoid gaps in the list. Here's the output with and without the idiom in case of role sap-hana-preconfigure in RHEL 7 - see https://github.com/linux-system-roles/sap-hana-preconfigure/blob/f41334b41de06a92741d09796e9f412537660a45/vars/RedHat_7.yml#L3 :

TASK [sap-hana-preconfigure : list of required SAP Notes] *************************************************************************************
ok: [lu0530v1] => {
    "__sap_hana_preconfigure_sapnotes | difference([''])": [
        "2009879",
        "2292690",
        "2455582",
        "2382421"
    ]
}

and here without it:
TASK [sap-hana-preconfigure : list of required SAP Notes] *************************************************************************************
ok: [lu0530v1] => {
    "__sap_hana_preconfigure_sapnotes": [
        "2009879",
        "",
        "2292690",
        "2455582",
        "2382421"
    ]
}